### DISCUSSION TITLE
"Downloading metadata from 0 peers" for newly-created torrent

### DISCUSSION THREAD

**Discussion by tiotrom** [transmission/transmission#2451] [Q&A] [answered]
I was able to create new torrents, but now every-time I create one, it stalls. 

![image](https://user-images.githubusercontent.com/34804533/92754741-6f357700-f38b-11ea-86c6-c16934368be2.png)

It has been like that for the past few months I'd say. Any idea why? And how to debug it? I properly add trackers and all that, but it simply does not work.

Cheers!

**Comment by ckerr** [comment, marked as answer]
It looks like this is a torrent that you yourself created, and then added it to your Transmission session as a magnet link?

If that's the case, try removing the magnet link and adding the `.torrent` file instead. That already has the metainfo and so you won't have to wait to download it from other peers. This is especially important if you created the torrent yourself; since if you've just created it, nobody else is going to have the metainfo yet and you'll stay blocked at this stage forever

**Reply by tio-trom** [reply]
Yes this is true. Seems to be working now. You can close this issue. Cheers!
